scripts/create-sentry-release

```diff
@@ -26,27 +30,24 @@ if [ -z "${SENTRY_ORG:-}" ] || [ -z "${SENTRY_PROJECT:-}" ]; then
   echo 'No SENTRY_ORG or SENTRY_PROJECT provided' && exit 1
 fi
 
-echo 'Pulling debug information from relay image...'
-docker pull "${DOCKER_IMAGE}"
-
-docker run --rm --entrypoint cat "${DOCKER_IMAGE}" /opt/relay-debug.zip > relay-debug.zip
-docker run --rm --entrypoint cat "${DOCKER_IMAGE}" /opt/relay.src.zip > relay.src.zip
-docker run --rm --entrypoint tar "${DOCKER_IMAGE}" -cf - /lib/x86_64-linux-gnu > libs.tar
-tar xf libs.tar
+echo 'Downloading debug info, source bundle, system symbols...'
+gsutil cp \
+  "gs://dicd-team-devinfra-cd--relay/release-assets/${REVISION}/relay-debug.zip" \
+  "gs://dicd-team-devinfra-cd--relay/release-assets/${REVISION}/relay.src.zip" \
+  "gs://dicd-team-devinfra-cd--relay/release-assets/${REVISION}/libs.tar" \
+  .
 
 echo 'Uploading debug information and source bundle...'
 sentry-cli upload-dif ./relay-debug.zip ./relay.src.zip
 
 echo 'Uploading system symbols...'
+tar xf libs.tar
 sentry-cli upload-dif lib/x86_64-linux-gnu
 
-echo 'Obtaining version information...'
-VERSION="$(docker run --rm "${DOCKER_IMAGE}" --version | cut -d' ' -f2)"
-RELEASE="relay@${VERSION}+${REVISION}"  # match RELEASE in relay_log::setup
-
 echo "Creating a new release and deploy for ${RELEASE} in Sentry..."
+RELEASE="relay@${REVISION}"  # match RELEASE in relay_log::setup
 sentry-cli releases new "${RELEASE}"
 sentry-cli releases set-commits "${RELEASE}" --commit "${GITHUB_PROJECT}@${REVISION}"
-sentry-cli releases deploys "${RELEASE}" new -e release
+sentry-cli releases deploys "${RELEASE}" new -e "$ENVIRONMENT"
```

Comment:
I'm not sure this actually makes sense. Usually, canary is a "deployment" for us, but not an "environment", so I need a bit of input on how to read the above gocd configuration. 

In freight, we used to have a "canary" pod that was part of a regular production deployment. 99% of the time, it would be no different than all others, and we want all sentry data emitted from that pod to be labeled as "production" environment. Otherwise, we don't see it in Sentry with default filters. 

When we selected a "canary" deployment in Freight, this one pod would be changed to the new revision. We didn't care about the environment, because it had a new release name anyway. 

The next production deployment reset the canary, too.

How does the current pipeline compare to this?

Reply:
To clarify on the terminology used here: Our "canary" deployment isn't a canary in the sense that devinfra has been using it. Instead, it's an experimental deployment/tier that receives an equal share of traffic.

Reply:
Ah... okay. So in this case, I'll create a separate deploy-relay-experimental pipeline that will just deploy the "canary".

And then the regular deployment pipeline will deploy that canary pod too.

Reply:
As for sentry releases - it doesn't seem to be working as you describe it? I expected a new release name for canary deploys but they're the same.
Example:

https://freight.getsentry.net/deploys/relay/canary/131
`Created release relay@23.3.0+0d52aad0aa46887b6009db52d5d037e625b51975.`

https://freight.getsentry.net/deploys/relay/production/711
`Created release relay@23.3.0+0d52aad0aa46887b6009db52d5d037e625b51975.`

https://sentry.my.sentry.io/organizations/sentry/releases/relay%4023.3.0%2B0d52aad0aa46887b6009db52d5d037e625b51975/?project=4

Reply:
Canary deploys should have the same release as their regular deploys. The release name purely depends on the build.

Reply:
Ok, same release name then. How about environment? I made a new deploy-relay-experimental pipeline and right now it's `experimental` environment: https://github.com/getsentry/relay/pull/1724/files#diff-4717184ee67f6e9554253d98f87bbbfc8c64255329f47245d42249035c106939R65

Should it be `experimental` or `production`?

Reply:
The environment can be `production` in both cases, that's what it is. We differentiate purely by the release name (version+sha).